src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinFeature.kt

```diff
@@ -58,7 +58,15 @@ enum class KotlinFeature(private val enabledByDefault: Boolean) {
      * In addition, the adjustment of behavior using get:JvmName is disabled.
      * Note also that this feature does not apply to setters.
      */
-    KotlinPropertyNameAsImplicitName(enabledByDefault = false);
+    KotlinPropertyNameAsImplicitName(enabledByDefault = false),
+
+    /**
+     * This feature represents whether to handle [kotlin.time.Duration] using [java.time.Duration] as conversion bridge.
+     *
+     * This allows use Kotlin Duration type with [com.fasterxml.jackson.datatype.jsr310.JavaTimeModule].
+     * `@JsonFormat` annotations need to be declared either on getter using `@get:JsonFormat` or field using `@field:JsonFormat`.
```

Comment:
I would like to add a reference to `[jackson-module-kotlin#651]` which summarizes this issue.

```suggestion
     * `@JsonFormat` annotations need to be declared either on getter using `@get:JsonFormat` or field using `@field:JsonFormat`.
     * See [jackson-module-kotlin#651] for details.
```